### PR TITLE
Write Cloud Functions entry function for invoking InfoCompiler by HTTP trigger, set dependencies and configurations, and write instructions in README.

### PR DESCRIPTION
**Pull request by KathyFeiyang**
### What is a quick description of the change?

**_Write Cloud Functions entry function for invoking InfoCompiler by HTTP trigger, set dependencies and configurations, and write instructions in README._**
1. Write Could Functions entry point `InfoCompilerFunction.service`. It will be triggered by HTTP requests.
2. Experiment with Cloud Functions and Cloud Scheduler service configurations for executing InfoCompiler and triggering it according to schedule.
3. Add dependencies to access Cloud Functions primitives and add a plugin to execute Cloud Functions locally.
4. Write instructions for how to use `gcloud` commands to set up Cloud Functions and Cloud Scheduler services in the [README](https://github.com/googleinterns/voter-central/tree/kathy_cloud_functions_cloud_scheduler/info-compiler), such as setting the valid service accounts.
image

### Is this fixing an issue?

none

### Are there more details that are relevant?

none

### Check lists (check `x` in `[ ]` of list items)

- [ N/A ] Test written/updating
- [ N/A ] Tests passing
- [ x ] Coding style (indentation, etc)

Please explain why any are not present, if any.

### CODE REVIEW

**Review by kellyktan — APPROVED** [review]
Have you run this already? How'd it go?

**Comment by KathyFeiyang** [comment]
> Have you run this already? How'd it go?

It worked! It stored entities in the database in the same way as when I was executing InfoCompiler on my virtual machine. I also tried setting Cloud Scheduler to a specific time and waited to see if it triggered Cloud Functions correctly. That part works as well, as long as authentication is set up correctly as outlined in the README.
Moving forward, I will definitely conduct additional tests to make sure of this.

File: info-compiler/src/main/java/com/google/sps/infocompiler/InfoCompilerFunction.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import java.io.IOException;
+
+/**
+  * The entry point of Google Cloud Functions: Compiles location-specific information for
+  * elections, positions and candidates.
+  */
+public class InfoCompilerFunction implements HttpFunction {
+  @Override
+  public void service(HttpRequest request, HttpResponse response) 
+      throws IOException { // The full package is used here to prevent naming conflicts.
```
Comment (suparnabarua) [inline info-compiler/src/main/java/com/google/sps/infocompiler/InfoCompilerFunction.java]: What full package?

**Review by suparnabarua — APPROVED** [review]

File: info-compiler/src/main/java/com/google/sps/infocompiler/InfoCompilerFunction.java (hunk 1 above)
Comment (KathyFeiyang) [reply info-compiler/src/main/java/com/google/sps/infocompiler/InfoCompilerFunction.java]: Sorry this was fixed!

**Review by tjyoung99 — APPROVED** [review]

**Review by ganewman — APPROVED** [review]

**Pull request merged**